mono/profiler/mono-profiler-log.c

```diff
@@ -5159,6 +5160,7 @@ mono_profiler_startup (const char *desc)
 		}
 		if ((opt = match_option (p, "heapshot", &val)) != p) {
 			events &= ~MONO_PROFILE_ALLOCATIONS;
+			events &= ~MONO_PROFILE_GC_MOVES;
```

Comment:
> The serve no purpose when allocation profiling is disabled.

Oops.

> More importantly, have you discussed this change with the profiler team?

AFAIK, they always enable allocation events anyway, so this shouldn't affect them. But I'll check with Moya to be sure.